runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerConfig.java

```diff
@@ -32,87 +32,87 @@
 public interface PrayerConfig extends Config
 {
 	@ConfigItem(
-		position = 0,
-		keyName = "prayerflick",
-		name = "Prayer flick helper",
-		description = "Enable the prayer flick helper"
+			position = 0,
```

Comment:
Do not reindent the fields here, see https://github.com/runelite/runelite/wiki/Code-Conventions